package.json

```diff
@@ -56,7 +56,7 @@
     },
     "dependencies": {
         "@babel/runtime": "^7.12.5",
-        "@matrix-org/analytics-events": "^0.3.0",
+        "@matrix-org/analytics-events": "^0.4.0",
```

Comment:
this change appears unrelated

Reply:
It's needed because I added Favourite Messages to WebScreen in that version (https://github.com/matrix-org/matrix-analytics-events/commit/4ef88563c8dc3f5a5bd8ed1194511ab597c98b7b).

Reply:
That constant doesn't appear to be used though? Or I'm not seeing the magic that glues screens to analytics-events.